lib/Backend/GlobOpt.cpp

```diff
@@ -9585,6 +9584,32 @@ GlobOpt::TypeSpecializeInlineBuiltInBinary(IR::Instr **pInstr, Value *src1Val, V
             break;
         }
 
+        case Js::OpCode::InlineMathPow:
+        {
+#ifndef _M_ARM
```

Comment:
I would rather that we do the optimization for all architectures together to maintain parity for ARM. ARM has a way of lagging behind when we defer ARM specific stuff for later.